### PR TITLE
build using sbt 1.x

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: project/build.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #Activator-generated Properties
 #Sat Sep 03 14:50:21 AEST 2016
 template.uuid=79abc40b-9334-4daa-972c-6bfc4c9c90e4
-sbt.version=0.13.16
+sbt.version=1.1.4
```
Comment (ignasi35) [inline project/build.properties]: `1.1.4`? Wow! 👍

Comment (renatocaval) [reply project/build.properties]: #YOLO

**Comment by ignasi35** [comment]
@renatocaval This build breaks somehow. We should have a look at it.

**Comment by ignasi35** [comment]
hey @renatocaval this build is still broken in Travis for some reason. :-(
I've just re-run it.

**Comment by ignasi35** [comment]
Just confirmed locally the issue of this build is on `.travis.yml`: uses `sbt +test` instead of `sbt test` as @TimMoore pointed out.

**Comment by renatocaval** [comment]
Travis is kind of broken for this PR. We have two builds now and one is failing. 

This happened because I pushed my last commit to `upstream\sbt-1.x` instead of `renatocaval\sbt-1.x`. After realising the mistake I removed `upstream\sbt-1.x`, but in the meantime Travis started to build it and somehow got confused about which branch is related to this PR.

**Comment by renatocaval** [comment]
I just re-pushed the same changes to `upstream\sbt-1.x` and Travis picked it again. The build should pass now. After that this can be merged, ie: `renatocaval\sbt-1.x` -> `online-auction-scala\master`.

I will delete `upstream\sbt-1.x` afterwards.

**Comment by ignasi35** [comment]
Build pass. Merge the thing! :-P

**Pull request merged**